recipes-extended/ostree/ostree/0001-ostree-fetcher-curl-set-a-timeout-for-an-overall-req.patch

```diff
@@ -4,6 +4,8 @@ Date: Sat, 3 Jul 2021 20:37:08 -0300
 Subject: [PATCH] ostree-fetcher-curl: set a timeout for an overall request
  processing
 
+Upstream-Status: Inappropriate [embedded specific]
+
```

Comment:
I will propose a drop of this one in  https://github.com/uptane/meta-updater/pull/119 but not yet